api/v1/model/src/commonMain/kotlin/Repository.kt

```diff
@@ -33,7 +39,16 @@ data class Repository(
 
     /** The url to the repository. */
     val url: String
-)
+) {
+    companion object {
+        const val INVALID_URL_MESSAGE = "The entity url is malformed."
```

Comment:
Not sure whether these constants and functions should be public; they then become part of the official API. Therefore, it is probably better to have them as private members of the file.

Reply:
To maintain consistency with the other data classes, I made these variables and functions public. Do you think we should change their visibility to private as well?

Reply:
Good point. No strong feelings about it, but I would tend to say yes. The public API should be as small as possible.

Reply:
Ok, probably not our most pressing issue currently.